File: scratch/check_schema.js
const { createClient } = require('@supabase/supabase-js');
const fs = require('fs');

let supabase_url = process.env.NEXT_PUBLIC_SUPABASE_URL;
let supabase_key = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY;

if (!supabase_url || !supabase_key) {
  const env = fs.readFileSync('.env.local', 'utf-8');
  for (const line of env.split('\n')) {
    if (line.includes('=')) {
      const [k, v] = line.trim().split('=');
      const cleanVal = v.replace(/['"]/g, '').trim();
      if (k === 'NEXT_PUBLIC_SUPABASE_URL') supabase_url = cleanVal;
      if (k === 'NEXT_PUBLIC_SUPABASE_ANON_KEY') supabase_key = cleanVal;
    }
  }
}

const supabase = createClient(supabase_url, supabase_key);

async function check() {
  const { data, error } = await supabase.from('production_plan_supplementary').select('*').limit(1);
  if (error) {
    console.error("Error:", error);
  } else {
    console.log("Data sample:", data);
  }
}

check();

File: scratch/check_schema.py
import os
from supabase import create_client

supabase_url = os.environ.get("NEXT_PUBLIC_SUPABASE_URL")
supabase_key = os.environ.get("NEXT_PUBLIC_SUPABASE_ANON_KEY")

if not supabase_url or not supabase_key:
    # Read from .env.local
    with open(".env.local", "r") as f:
        for line in f:
            if "=" in line:
                k, v = line.strip().split("=", 1)
                if k == "NEXT_PUBLIC_SUPABASE_URL":
                    supabase_url = v.strip('"').strip("'")
                elif k == "NEXT_PUBLIC_SUPABASE_ANON_KEY":
                    supabase_key = v.strip('"').strip("'")

client = create_client(supabase_url, supabase_key)
# Let's get one row to inspect fields
res = client.table("production_plan_supplementary").select("*").limit(1).execute()
print("Data sample:", res.data)
